Sources/ThresholdKey/Encryption.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  File.swift
+//  
+//
+//  Created by CW Lee on 12/09/2023.
+//
+
+import Foundation
+#if canImport(lib)
+    import lib
+#endif
+
+// secp256k1 curve
+public let secpN = "fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141"
+
+
```

Comment:
Why are we directly exposing the encrypt and decrypt methods here?

All the logic around this and setting the relevant general store domain should be handled internally, no?